edelphi/src/main/java/fi/metatavu/edelphi/utils/comments/AbstractReportPageCommentProcessor.java

```diff
@@ -56,6 +62,28 @@ public QueryPage getQueryPage() {
     return queryPage;
   }
   
+  /**
+   * Returns comment label as string
+   * 
+   * @param id comment id
+   * @return comment label as string
+   */
+  public String getCommentLabel(Long id) {
+    Map<String, String> valueMap = answers.get(id);
+    if (valueMap != null && !valueMap.isEmpty()) {
+      Set<Entry<String,String>> entrySet = valueMap.entrySet();
+      List<String> labels = entrySet.stream()
+        .map((entry) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "entry" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)